mamba_gator/envmanager.py

```diff
@@ -596,6 +599,25 @@ async def list_available(self) -> Dict[str, List[Dict[str, str]]]:
             # dictionary with error info
             return data
 
+        def process_mamba_repoquery_output(data: Dict) -> Dict:
+            """Make a dictionary with keys as packages name and values
+            containing the list of available packages to match the json output
+            of "conda search --json".
+            """
+
+            data_ = {}
+            for entry in data['result']['pkgs']:
+                name = entry.get('name')
+                if name in data_.keys():
+                    data_[name].append(entry)
+                else:
+                    data_[name] = [entry]
```

Comment:
You could simplify this like this:

```suggestion
            data_ = collections.defaultdict(lambda : [])
            for entry in data['result']['pkgs']:
                name = entry.get('name')
                if name is not None:
                    data_[name].append(entry)
```

Under the hood the default dict will create an empty list for data_[name] if name key does not exist.

Reply:
Indeed, this is more clear, done!